### PR TITLE
docs: Add v0 import example to v0-compatibility

### PR DESCRIPTION
**Pull request by charlieegan3**
This shows another way to use two versions of rego in the same app.

### CODE REVIEW

File: docs/content/v0-compatibility.md [hunk 1]
```diff
@@ -111,6 +114,47 @@ r := rego.New(
...
+```go
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	v0rego "github.com/open-policy-agent/opa/rego"
+	// v1rego "github.com/open-policy-agent/opa/v1/rego"
```
Comment (johanfylling) [inline docs/content/v0-compatibility.md]: This is for demonstrative purposes?

Maybe consider a more informative comment, e.g.
```suggestion
	"github.com/open-policy-agent/opa/rego" // v0 Rego API
```

Maybe we could also drop the `v0rego` alias as that's not necessary for disambiguation.

**Review by johanfylling — APPROVED** [review]
👍

File: docs/content/v0-compatibility.md (hunk 1 above)
Comment (charlieegan3) [reply docs/content/v0-compatibility.md]: 👍 yeah that makes sense.

File: docs/content/v0-compatibility.md [hunk 2]
```diff
@@ -111,6 +114,47 @@ r := rego.New(
...
+```go
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/open-policy-agent/opa/rego"
+	// v1rego "github.com/open-policy-agent/opa/v1/rego"
```
Comment (ashutosh-narkar) [inline docs/content/v0-compatibility.md]: Did you intend to keep this commented out line for informational purposes or something?

Comment (charlieegan3) [reply docs/content/v0-compatibility.md]: Yeah it was intentional but I've made that clearer in https://github.com/ashutosh-narkar/opa/pull/8/commits/d6db4ac82d9e56ce534cd654e64f24c27f90bee0

**Review by ashutosh-narkar — APPROVED** [review]

**Pull request merged**